specs/recipient_spec.rb

```diff
@@ -0,0 +1 @@
+require "test_helper"
```

Comment:
This should be `require_relative`, not `require`.  Using `require` here breaks `rake` (but not `guard`).

```suggestion
require_relative "test_helper"
```

Reply:
Also, while it's hard to test an abstract class it would be good to at a minimum make sure that your template methods `raise` `NotImplementedError`.